### PR TITLE
feat: email verification

### PR DESCRIPTION
**Pull request by evgeniy-ac**

### CODE REVIEW

File: Directory.Packages.props [hunk 1]
```diff
@@ -216,4 +217,4 @@
     <PackageVersion Include="RestEase.SourceGenerator" Version="1.6.4" PrivateAssets="All" IncludeAssets="runtime; build; native; contentfiles; analyzers" />
     <PackageVersion Include="Stl.Generators" Version="$(StlFusionVersion)" PrivateAssets="All" IncludeAssets="runtime; build; native; contentfiles; analyzers" />
   </ItemGroup>
-</Project>
+</Project>
```
Comment (frolyo) [inline Directory.Packages.props]: accidental?

File: src/dotnet/Users.Service/Email/EmailSender.cs [hunk 2]
```diff
@@ -0,0 +1,33 @@
+using ActualChat.Users.Module;
+using MailKit.Net.Smtp;
+using MimeKit;
+
+namespace ActualChat.Users.Email;
+
+public interface IEmailSender
```
Comment (frolyo) [inline src/dotnet/Users.Service/Email/EmailSender.cs:7]: Afair we have SmsGateway, so maybe call then consistently - both Sender or both Gateway...

**Review by frolyo — APPROVED** [review]

**Review by alexis-kochetov — APPROVED** [review]

File: src/dotnet/Users.Service/Email/EmailSender.cs (hunk 2 above)
Comment (evgeniy-ac) [reply src/dotnet/Users.Service/Email/EmailSender.cs:7]: Actually you don't have `SmsGateway`. It's `ITextMessageGateway` and IMO it's too abstract and its name conflicts with a `Gateway` pattern. `SmsSender` is more clear.

File: Directory.Packages.props (hunk 1 above)
Comment (evgeniy-ac) [reply Directory.Packages.props]: done

**Pull request merged**